api/src/main/resources/application-beta.yml

```diff
@@ -13,4 +13,7 @@ spring:
 
 paypal:
   clientId: AVbr-mNnSiaEKeAa97MPVerIi_G18U8GMqq8XyjXvAfRKH2bLhMy_Ue3XOtdNyjWScSalu8JoN3YJrnP
-  mode: sandbox
\ No newline at end of file
+  mode: sandbox
+
+twitch:
+  whiteListedAccounts: nckackerman, M1sterc001guy, vgame18
```

Comment:
We might not want to check this into source control. Would it be difficult to keep the whitelist and blacklist on S3 so only we can see it?

Reply:
Good callout! This doesnt need to be source controlled and shouldnt be. Ive moved this list to the S3 secrets bucket.